reaktive/src/commonTest/kotlin/com/badoo/reaktive/completable/CompletableToCompletableTests.kt

```diff
@@ -23,9 +23,6 @@ class CompletableToCompletableTestsImpl(
     transform: Completable.() -> Completable
 ) : CompletableToCompletableTests, SourceTests by SourceTestsImpl(TestCompletable(), { transform().test() }) {
 
-    @Deprecated("Just to fix complilation issues")
-    constructor() : this({ throw UnsupportedOperationException() })
```

Comment:
Hmm, it's not part of the public API. But there were compilation errors a while ago. Does it compile across all targets?

Reply:
Not sure, let me check. The JVM was compiling fine.